CHANGELOG.md

```diff
@@ -42,6 +46,7 @@
 ### Changed
 - Updated gap and styles on Row component
 
+[0.66.2]: https://github.com/marshmallow-insurance/smores-react/compare/v0.66.1...v0.67.0
```

Comment:
this should be `0.67.0` or since it's a small change probably better to patch it as 0.66.2